### PR TITLE
Added project call and fixed wrong url for october meetup page

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: content/news/2023-11-29-call-for-projects.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Please provide a one page description that includes:
+- a general outline of your project and the software you plan to develop.
+- how you plan to fund and staff the development work.
+- if there are any RSE groups or other software development units at your institution you have access to. 
+- the timeline of your project.
+- the composition of your team.
+- the work that has been done so far towards developing the software for your project such as any prototypes developed or technical plans drafted.
+- any specific requirements or context a consultant would need to advise on the technical implementation plan for your project.
+- expected impact or value of an external RSE consultant for your project.
+
```
Comment (rlskoeser) [inline content/news/2023-11-29-call-for-projects.md:24]: this is pretty nit-picky, but I think we should either start with capitals or drop the periods at the end of each bullet point (it looks weird to me as is). I vote for dropping the periods

File: content/news/2023-09-28-announcing-october-meetup.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ author: DHTech
 thumbnail: /images/posts/meetup.jpeg
 featureImage: /images/posts/meetup.jpeg
 featureImageCap: Photo by <a href="https://unsplash.com/@charlesdeluvio?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">charlesdeluvio</a> on <a href="https://unsplash.com/photos/wn7dOzUh3Rs?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
-
+url: DHTech-October-Meetup
```
Comment (rlskoeser) [inline content/news/2023-09-28-announcing-october-meetup.md:9]: if we need the top level url for some reason and an alias is acceptable:

```suggestion
aliases: 
- /DHTech-October-Meetup/
```

**Review by rlskoeser — APPROVED** [review]
content looks good, minor suggestions

File: content/news/2023-09-28-announcing-october-meetup.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jdamerow) [reply content/news/2023-09-28-announcing-october-meetup.md:9]: hm, it actually works just fine on the website (for some reason it doesn't work locally for me). I guess I can just remove this line then.

File: content/news/2023-11-29-call-for-projects.md (hunk 1 above)
Comment (jdamerow) [reply content/news/2023-11-29-call-for-projects.md:24]: 😆 I was actually thinking I should capitalize the first letter but then was too lazy. I vote for capitalizing over removing periods (no period at the end of such long sentences seems weird to me) but either works.

Comment (rlskoeser) [reply content/news/2023-11-29-call-for-projects.md:24]: capitalizing works! I guess the lower case is probably too informal

**Pull request merged**